outlook/src/taskpane/components/Login/Login.tsx

```diff
@@ -136,8 +240,55 @@ class Login extends React.Component<{}, LoginState> {
                     <div>Search and store insights on your contacts.</div>
                 </div>
             </div>
-        </>
-    );
+        </>);
+    }
+
+    /**
+     * Check the "urlError" state and return the appropriate error message.
+     */
+    _getErrorMessage = () => {
```

Comment:
I somehow feel like all these errors should link the documentation?
End users might be lost with something simply showing: "Please accept the permission.".

Reply:
Ok, so I can simplify a lot the code if we always display the FAQ link

Reply:
Maybe double-check with LNA first.
And we could have a different explanation for each, something like.

> It looks like you did not give the Outlook module the appropriate Odoo permissions.
> Please refer to `<doc link>` for more information.

Reply:
I'll see during the testing BE (it would not be difficult to change)